thunder/dynamo/__init__.py

```diff
@@ -1,4 +1,3 @@
 from thunder.dynamo.compiler import ThunderCompiler, thunderfx
 
-
```

Comment:
This change is unnecessary.

Reply:
ack, thanks, fixed.

rewrote history to do so, though; note the diff is respun. sorry about that.